### PR TITLE
WIP: Feature - file upload

### PR DESCRIPTION
**Pull request by landongerrits**
screenshot 2019-01-24 08 39 35

### CODE REVIEW

File: src/Components/FileUpload/FileUpload.js [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+    this.state = {
+      file: null
+    };
+  }
+
+  submitFile = (event) => {
+    event.preventDefault();
+    const formData = new FormData();
+    formData.append('file', this.state.file[0]);
+    axios.get('/upload', {
```
Comment (mattyarmolich) [inline src/Components/FileUpload/FileUpload.js]: post

File: src/Components/FileUpload/FileUpload.js [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+        params: {
+            // hard coded test params for now
+            file_name: "Basic_Stats.csv",
```
Comment (mattyarmolich) [inline src/Components/FileUpload/FileUpload.js]: pass file: formData to send your file

File: src/Components/FileUpload/FileUpload.js [hunk 3]
```diff
@@ -0,0 +1,47 @@
...
+            user_id: 12345,
+            bucket_name: "uploads" | "classified"
+        }
+      })
+      .then(function (response) {
+        console.log(response);
```
Comment (mattyarmolich) [inline src/Components/FileUpload/FileUpload.js]: sucessful upload

File: src/Components/FileUpload/FileUpload.js [hunk 4]
```diff
@@ -0,0 +1,47 @@
...
+    super();
+    this.state = {
+      file: null
+    };
+  }
+
+  submitFile = (event) => {
+    event.preventDefault();
+    const formData = new FormData();
+    formData.append('file', this.state.file[0]);
```
Comment (mattyarmolich) [inline src/Components/FileUpload/FileUpload.js]: iterate through using.map for each form data object

File: src/Components/FileUpload/FileUpload.js (hunk 2 above)
Comment (mattyarmolich) [reply src/Components/FileUpload/FileUpload.js]: also maybe use a for to do for different formdatas

**Comment by landongerrits** [comment]
drag n drop is working. Already merged into master, just working on CSS styling

Closing this PR

**Pull request closed without merging**